onadata/apps/logger/tests/models/test_version.py

```diff
@@ -0,0 +1,39 @@
+from django.test import TestCase
+
+from onadata.apps.logger.models import VersionTree
+
+
+class VersionTreeTestCase(TestCase):
```

Comment:
since you defined VersionTreeException, and assuming it will stay in code - I'd like to see a test where .get_root_path raises it